src/lib/transform.js

```diff
@@ -61,6 +72,13 @@ function transformSelector(value, options, result, item) {
   }).processSync(value);
 }
 
+
+/** 
+ * @param {any} node
```

Comment:
The types of `node` and `property` are linked, so `property` will always be a valid key for `node`. Tried to create a dependent type but it looked a bit too much.  It's probably better to refactor since the function is called only once.